extensions/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCore.csproj

```diff
@@ -0,0 +1,46 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <RollForward>LatestMajor</RollForward>
+        <AssemblyName>Microsoft.KernelMemory.MemoryDb.AzureCosmosDBMongoVCore</AssemblyName>
```

Comment:
same here and other places, I think the name is missing a `DB` after Mongo. 

For instance this one would be `Microsoft.KernelMemory.MemoryDb.AzureCosmosDBMongoDBVCore`.

Same for namespace and class names

Reply:
Updated the name